components/TextInputWithLabel.tsx

```diff
@@ -12,7 +12,7 @@ export default function TextInputWithLabel({
 }) {
     const [wordCount, setWordCount] = React.useState(0);
     return (
-        <View>
+        <View {...inputProps}>
```

Comment:
Ik denk dat dit niet verstandig is, nu krijgt de wrapper precies dezelfde styling als de text input die er in staat. Dat is voor nu alleen flex-grow en de height, maar dat voelt alsof het later problemen kan opleveren.